tests/integration/test_records.py

```diff
@@ -56,29 +58,37 @@ def test_float_vector(client):
     ensemble_id = _create_ensemble(client)
 
     records = [
-        ("name", "[1, 2, 3, 4, 5]", True),
-        ("name", "[5, 4, 3, 2, 1]", False),
-        ("fail", "yes", False),
+        ("name", "[1, 2, 3, 4, 5]", True, status.HTTP_200_OK),
```

Comment:
If we're writing the `status_code`, perhaps we can just implicitly assume that when `status.HTTP_200_OK` then `should_succeed = True`?

Reply:
Will refactor it a bit.